host_vars/gub37-core/base.yml

```diff
@@ -5,3 +5,4 @@ role: corerouter
 model: "avm_fritzbox-7530"
 
 wireless_profile: freifunk_default
+bandsteering: true
```

Comment:
I would not call it bandsteering. Depending on DAWN configuration it does several things, for example, client load balancing.

Reply:
Calling it dawn felt to specific. Do you have a suggestion?